torch/_inductor/simple_fsdp.py

```diff
@@ -122,7 +117,7 @@ def reorder_reduce_scatter(
 
 
 def get_node_type(node, prev_nodes=None) -> int:
-    # node_type: {0: all gather; 1: wait_tensor; 2: computation; 3: reduce scatter; 4: convert_element_type after rs}
+    # node_type: {0: all gather; 1: wait_tensor; 2: computation; 3: reduce scatter;}
```

Comment:
Let's remove this comment; otherwise we need to maintain it if the NodeType changes.